modules/persistent-storage-csi-efs-troubleshooting.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+
+[id="efs-troubleshooting_{context}"]
+= AWS EFS troubleshooting
+
+The following information provides guidance on how to troubleshoot issues with AWS EFS:
+
+* The AWS EFS Operator and CSI driver run in namespace `openshift-cluster-csi-drivers`.
+
+* To initiate gathering of logs of the AWS EFS Operator and CSI driver, run the following command:
++
+[source, screen]
+----
+$ oc adm must-gather
+[must-gather      ] OUT Using must-gather plugin-in image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:125f183d13601537ff15b3239df95d47f0a604da2847b561151fedd699f5e3a5
+[must-gather      ] OUT namespace/openshift-must-gather-xm4wq created
+[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-2bd8x created
+[must-gather      ] OUT pod for plug-in image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:125f183d13601537ff15b3239df95d47f0a604da2847b561151fedd699f5e3a5 created
+----
+
+* To show AWS EFS Operator errors, view the `ClusterCSIDriver` status:
++
+[source, screen]
```

Comment:
```suggestion
[source,terminal]
```

We don't use `[source,screen]`. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output